programs/create-genesis/golos_objects.hpp

```diff
@@ -92,6 +92,7 @@ struct account_object {
     uint16_t referrer_interest_rate;
     time_point_sec referral_end_date;
     asset referral_break_fee;
+    fc::optional<share_type> reputation;
```

Comment:
this makes previously used source state incompatible, propose update version number (Golos should also generate state-snapshot with updated version)